apps/wallet/src/ui/app/pages/home/index.tsx

```diff
@@ -71,7 +71,6 @@ const HomePage = ({ disableNavigation, limitToPopUpSize = true }: Props) => {
                     bottomNavEnabled={!disableNavigation}
                     dappStatusEnabled={!disableNavigation}
                     topNavMenuEnabled={!disableNavigation}
-                    centerLogo={true}
```

Comment:
I didn't see any screens on Figma where the logo was the only element present in the header and centered in the middle (as opposed to being on the left). If I need to add this back just let me know!

Reply:
Left alignment is what we want when there is only logo in the header @williamrobertson13 
Thank you Will.